src/mbgl/renderer/renderer_impl.cpp

```diff
@@ -384,6 +377,15 @@ void Renderer::Impl::render(const UpdateParameters& updateParameters) {
         }
     }
 
+    const auto encoder = parameters.context.createCommandEncoder();
```

Comment:
nit: `std::unique_ptr<gfx::CommandEncoder> encoder` would be more readable

Reply:
?

Reply:
I mean since the actual type is quite short `std::unique_ptr<gfx::CommandEncoder>`, might be worth disclosing it for readability instead of `auto encoder`.